ci/do_ci.sh

```diff
@@ -193,6 +194,7 @@ elif [[ "$1" == "cmake.exporter.otprotocol.test" ]]; then
   rm -rf *
   cmake -DCMAKE_BUILD_TYPE=Debug  \
         -DWITH_OTLP=ON \
+        -DWITH_OTLP_HTTP=ON \
```

Comment:
Do we need also add `-DWITH_EXAMPLES_HTTP=ON` in at least one CI job? Or we can not ensure the http example can be compiled success.